src/routes/safe/components/Transactions/TxsTable/ExpandedTx/TxDescription/index.jsx

```diff
@@ -105,22 +103,21 @@ const CustomDescription = ({
         {' '}
         to:
       </Bold>
-      <br />
       <EtherscanLink type="address" value={recipient} />
-    </Paragraph>
-    <Paragraph className={classes.txData} data-testid={TRANSACTIONS_DESC_CUSTOM_DATA_TEST_ID}>
+    </Block>
+    <Block className={classes.txData} data-testid={TRANSACTIONS_DESC_CUSTOM_DATA_TEST_ID}>
```

Comment:
same here

Reply:
It has the component `EtherscanLink`, which has the component `Block` as his root element